src/explorer/weights/WeightSlider.js

```diff
@@ -38,12 +69,43 @@ export class WeightSlider extends React.Component<Props> {
   }
 }
 
-function convertWeightToSliderValue(weight: number): number {
-  return weight === 0 ? WEIGHT_SLIDER_MIN : Math.log2(weight);
+/**
+ * Converts a Weight into a SliderPosition.
+ *
+ * If the Weight corresponds to an illegal slider value (i.e. weight < 0 or
+ * weight > 2**MAX_SLIDER), then an error will be thrown.
+ * be thrown.
+ *
+ * Over the domain of legal weights, `weightToSlider` and `sliderToWeight`
```

Comment:
```js
 * Over the domain of legal weights, `weightToSlider` and `sliderToWeight`
 * compose to form the identity function.
```

This doesn’t seem accurate. The weight (say) `0.01` is accepted by this
function and doesn’t throw, but also doesn’t roundtrip:

    > WS.sliderToWeight(WS.weightToSlider(0.01))
    0

Reply:
You're right. Updated the documentation.

Reply:
You're right. Updated the docs.